mail-filter/mailfilter/mailfilter-0.8.7.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Mailfilter is a utility to get rid of unwanted spam mails"
+HOMEPAGE="http://mailfilter.sourceforge.net/"
+SRC_URI="mirror://sourceforge/mailfilter/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86"
+IUSE="+ssl"
+
+DEPEND="sys-devel/flex
+	ssl? ( dev-libs/openssl:* )"
+RDEPEND=""
```

Comment:
This seems wrong:
1. RDEPEND too, surely
2. :* isn't likely to be correct if it links against it. Does it? (This affects 1.). Will need := instead.

Reply:
Yeah..not sure what happened here. I've changed to :0= and added openssl to RDEPEND.